app/models/vote.rb

```diff
@@ -1,14 +1,2 @@
 class Vote < ActiveRecord::Base
```

Comment:
There are other ways to do that. Eg. https://github.com/openSUSE/osem/blob/master/db/migrate/20140610165551_migrate_data_person_to_user.rb#L2